spec.html

```diff
@@ -120,6 +120,32 @@ <h1>
 		<emu-note type=editor>
 			In the case of an abrupt ~throw~ completion, the type of error to be created should match the type of the abrupt throw completion record. This could be revisited when merging into the main specification. Additionally, in the case of a ~break~ or ~continue~ completion, since those are not supported, a TypeError is expected.
 		</emu-note>
+		<emu-note>
+			Host environments may provide implementation-specific stack information for introspection APIs. If so, all Error objects originating from code executed inside a ShadowRealm must be subject to censorship. The specific steps are implementation-defined but must respect the following: The information exposed as part of Error's stack must contain only the details of the stack-frames associated to the ShadowRealm, and not reveal any details of frames associated with other realms, including the number of any such frames. Implementations can store information of the full stack if the error can cross into another Realm, however the implementation cannot use that information to reveal more information than allowed above.
```

Comment:
@mhofman @littledan check this note... and the one below...

Reply:
Editorial comment while I digest what this means: You can't have a note that says "must" in it--"normative notes" are not a thing. Just make this a paragraph and not a note.